src/interp/interp.h

```diff
@@ -305,13 +306,37 @@ struct LocalDesc {
   u32 end;
 };
 
+// Metadata for representing exception handlers associated with a function's
+// code. This is needed to look up exceptions from call frames from interpreter
+// instructions.
+struct CatchDesc {
+  Index tag_index;
+  u32 offset;
+};
+
+// The Try kind is for catch-less try blocks, which have a HandlerDesc that
```

Comment:
Is this comment out-of-date?  What is `Try kind` here?

Reply:
Thanks for catching this, yeah it's out-of-date now. I'll more clearly indicate what's included in the `Catch` case.